src/features/basic/show-space-remaining.ts

```diff
@@ -0,0 +1,68 @@
+import { shipsStore } from '@src/infrastructure/prun-api/data/ships';
+import { storagesStore } from '@src/infrastructure/prun-api/data/storage';
+import { watchEffectWhileNodeAlive } from '@src/utils/watch';
+
+function formatFixed(f: number) {
+  const x = f % 1 ? f.toFixed(2) : f;
+  return x.toString().replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ',');
+}
+
+function getStore(tile: PrunTile) {
+  const storeId =
+    tile.command.startsWith('SHPI') && tile.parameter
+      ? shipsStore.getByRegistration(tile.parameter)?.idShipStore
+      : tile.parameter;
+  if (!storeId) {
+    return;
+  }
+  return storagesStore.getById(storeId);
+}
+
+function onTileReady(tile: PrunTile) {
+  subscribe($$(tile.anchor, C.StoreView.column), column => {
+    const capacities = _$$(column, C.StoreView.capacity);
+    if (capacities.length < 2) {
+      return;
+    }
+    const weightIndex = capacities.findIndex(x => x.textContent?.includes('Weight'));
+    const volumeIndex = capacities.findIndex(x => x.textContent?.includes('Volume'));
```

Comment:
Try to avoid using localized text; otherwise, a feature will only work in English localization. Here, either access elements by index, or use localized labels from `PrunI18N` to search for an element. I recommend the former option, for simplicity.

Reply:
And, instead of `findIndex` use `find` - it will give you an element directly:

```suggestion
    const weight = capacities.find(x => x.textContent?.includes('Weight'));
    const volume = capacities.find(x => x.textContent?.includes('Volume'));
```